Makefile

```diff
@@ -604,23 +604,44 @@ endif
 # Packaging Themis Core: Linux distributions
 #
 
+ifeq ($(ENGINE),boringssl)
+ifeq ($(CRYPTO_ENGINE_LIB_PATH),)
+PACKAGE_EMBEDDED_BORINGSSL := yes
```

Comment:
so, if engine is boringssl, we'll pack boringssl as embedded library by default, right?

Reply:
If `ENGINE=boringssl` _**and**_ `ENGINE_LIB_PATH` is not set. That is, if the user has requested a build with BoringSSL from Themis submodule, not with some user-provided BoringSSL build from user-specified location.